frontend/src/modules/organization/components/view/organization-view-aside.vue

```diff
@@ -20,7 +20,7 @@
             <div class="flex gap-3 items-center">
               <app-platform :platform="identity.platform" custom-platform-icon-class="ri-community-fill" />
               <span class="text-gray-900 text-xs">
-                {{ (identity.url ? identity.url.split('/').at(-1) : identity.name) ?? getPlatformDetails(identity.platform)?.name }}</span>
+                {{ getPlatformDetails(identity.platform).organization.handle(identity) ?? getPlatformDetails(identity.platform)?.name }}</span>
```

Comment:
Same here, `getPlatformDetails(identity.platform)` could be undefined for custom platforms